testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/FilesResourceTest.java

```diff
@@ -71,13 +71,18 @@ public void schemaFile(@TestKeyspace CqlIdentifier keyspaceId) {
   }
 
   @Test
-  @DisplayName("Should return 404 if schema file coordinates are invalid")
+  @DisplayName("Should return 404 if schema file coordinates do not exist")
   public void schemaFileNotFound(@TestKeyspace CqlIdentifier keyspaceId) {
     CLIENT.expectSchemaFileStatus("unknown_keyspace", Status.NOT_FOUND);
-    CLIENT.expectSchemaFileStatus("malformed keyspace #!?", Status.NOT_FOUND);
```

Comment:
This was not covering the intended scenario: because the URL parameter was not encoded, it produced an invalid URL that was caught by Jersey and it would never actually hit the method in `FilesResource`.